docs/source/configuration/authorization.mdx

```diff
@@ -417,19 +417,18 @@ If _every_ requested field requires authentication and a request is unauthentica
 ### `@policy`
 
 The `@policy` directive marks fields and types as restricted based on authorization policies evaluated in a [Rhai script](../customizations/rhai/) or [coprocessor](../customizations/coprocessor). This enables custom authorization validation beyond authentication and scopes.
-The directive should include a `policies` argument that defines an array of the required policies.
+
+The `@policy` directive includes a `policies` argument that defines an array of the required policies. The following example shows a policy that requires all roles from the claims object to contain the string `"support"`:
```

Comment:
I think it'd be better here to use a generic policy string, not Rhai code (I'm not sure we should be suggesting Rhai code in general, more on that later). This paragraph & example make it seem like the default behavior of policies is to execute some code, which isn't the case.

Reply:
cc @Geal, for this and other comments from Dylan